app/upgrade_v_11_4_manifest.go

```diff
@@ -13,6 +13,7 @@ import (
 const manifestFilenameBase = "upgrade_manifest.json"
 
 type UpgradeManifest struct {
+	ConfigSource       string                     `json:"config_source"`
```

Comment:
Manifest must be deterministic (the same manifest data must be generated everywhere). As the implication, it must **\*not\*** contain data which are client specific, like for example file path.
All we can put here are sh256 hashes of the input files:
```suggestion
        // Following 2 hash data mambers are intentionally without `omitempty` parameter in `json:...` decorator
	GenesisFileSha256        string `json:"genesis_file_sha256"`
	NetworkConfigFileSha256  string `json:"network_config_file_sha256"`  
```